docs/resources/Guild.md

```diff
@@ -10,8 +10,8 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 |-------|------|-------------|
 | id | snowflake | guild id |
 | name | string | guild name (2-100 characters) |
-| icon | string | icon hash |
-| splash | string | splash hash |
+| icon | string | icon hash, [url format](#DOCS_CHANNEL/url-formatting) |
```

Comment:
Rather than adding more text, just link `icon hash` and `splash hash` etc to the docs section.